CHANGELOG.md

```diff
@@ -2,5 +2,10 @@
 
 ## 0.5.0
 - Compatible with .NET Standard 2.0
-- Quartz Facilities Quartz supports now Quartz.NET 3.0.2
-- BREAKING CHANGE: namespace is changed into Castle.Facilities.Quartz. Code remains the same
\ No newline at end of file
+- Quartz Facility supports now Quartz.NET 3.0.2
+- BREAKING CHANGE: namespace is changed into Castle.Facilities.Quartz. Code remains the same
+
+## 1.0.0
```

Comment:
The new version should be at the top of the file.

Reply:
moved